modules/core/src/main/java/org/apache/ignite/internal/thread/LogUncaughtExceptionHandler.java

```diff
@@ -33,6 +33,6 @@ public LogUncaughtExceptionHandler(IgniteLogger log) {
     /** {@inheritDoc} */
     @Override
     public void uncaughtException(Thread t, Throwable e) {
-        log.error("Uncaught exception in thread {}", e, t);
+        log.debug("Uncaught exception [thread={}]", e, t);
```

Comment:
not sure for that case we shouldn't use error log level

Reply:
ERROR requires immediate action. Is it OK to log it at WARN? The reason I decided to change this is because this handler is more like a stub. I believe that right way to handle the exception is to analyse it though some common failure handler, but not just put the line into log